.github/workflows/CI.yml

```diff
@@ -0,0 +1,78 @@
+name: CI
+# Run on master, any tag or any pull request
+on:
+  push:
+    branches: [master]
+    tags: ["*"]
+  pull_request:
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          # Test on the LTS, Invenia prod release, latest release
+          - 1.0.5
+          - 1.3
+          - 1
+        os:
```

Comment:
Maybe include a Mac build?

Reply:
The old travis file only tested on linux. @nickrobinson251 Is there anything in this repo that you think could be platform dependent?

Reply:
Don't see why we'd need multiple different OS. But 32bit could be useful for potential memory issues

Reply:
Hmm, looks like the doctests need to be fixed which seems outside the scope of this PR. I'm gonna leave the 32-bit option commented out with an issue that should be resolved.

Reply:
https://github.com/invenia/BlockDiagonals.jl/issues/44